CHANGELOG.adoc

```diff
@@ -85,6 +85,7 @@
 - https://github.com/eclipse-sirius/sirius-components/issues/1507[#1507] [diagram] Missing variables during execution of a source reconnection tool from the View description (otherEnd,semanticOtherEnd,edgeView and editingContext)
 - https://github.com/eclipse-sirius/sirius-components/issues/1521[#1521] [view] Canonical creation tools from contextual toolbar have no icons
 - https://github.com/eclipse-sirius/sirius-components/issues/1467[#1467] [layout] Elk is now able to compute a list layout. It is an internal change, and thus, nothing should change for the end user
+- https://github.com/eclipse-sirius/sirius-components/issues/1588[#1588] [tree][project][diagram] Add support for enabling/disabling the focus on a tree view element (that is not revealed yet) if a digram element is selected. Moved the action "New Model" and "Upload model" in a new tree toolbar.
```

Comment:
No need to add multiple tags, just pick the most relevant one, here [tree].

Reply:
Write each sentence on a new line. And here since you have multiple changes, you can use another tag since the move of the dialogs is technically a breaking change.

Reply:
On top of that, you have added this to the release notes of 2023.1.0, not 2023.3.0. This should be way higher on this file.